dali/util/cuda_utils.h

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) 2018, NVIDIA CORPORATION. All rights reserved.
```

Comment:
We can now move https://github.com/NVIDIA/DALI/blob/13136be77d90fabc6a412107e5a1d247037328f4/dali/error_handling.h#L35 
and (?)
https://github.com/NVIDIA/DALI/blob/13136be77d90fabc6a412107e5a1d247037328f4/dali/error_handling.h#L27
to this file.

Reply:
done